### ISSUE TITLE
Feature Request: Inbound filters

### ISSUE THREAD

**Issue by jianyuan** [jianyuan/terraform-provider-sentry#247]
Reference: https://docs.sentry.io/product/data-management-settings/filtering/

---

### Enhancement 1: New `custom_filters` property in the `sentry_project` resource

```terraform
resource "sentry_project" "my_project" {

  # ...

  custom_filters {
    blacklisted_ips = "127.0.0.1"
    release         = "1.*"
    error_messages  = "TypeError*"
  }
}
```

The nested properties refer to the properties in the custom filters section:

![CleanShot 2022-12-16 at 23 39 55](https://user-images.githubusercontent.com/103584/208208423-ef5d4943-c10c-46b5-8caf-0f8e3176c604.png)

---

### Enhancement 2: New `filters` property in the `sentry_project` resource

```terraform
resource "sentry_project" "my_project" {

  # ...

  filter {
    filter_id = "browser-extensions"
    active    = true
  }

  filter {
    filter_id = "localhost"
    active    = true
  }

  filter {
    filter_id  = "legacy-browsers"
    active     = true
    subfilters = [
      "ie10",
      "ie_pre_9",
      "opera_mini_pre_8",
      "safari_pre_6",
      "ie11",
      "android_pre_4",
      "ie9",
      "opera_pre_15"
    ]
  }

  filter {
    filter_id = "web-crawlers"
    active    = false
  }
}
```

The filters refer to this section:

![CleanShot 2022-12-16 at 23 48 59](https://user-images.githubusercontent.com/103584/208209319-a2fce59e-5b50-4809-b754-96b7639c4a2f.png)

**Comment by lpgarzonr** [comment]
looking forward for this feature

**Comment by jianyuan** [comment]
[This resource is now available in v0.12.0](https://registry.terraform.io/providers/jianyuan/sentry/latest/docs/resources/project_inbound_data_filter).

**Comment by andrewmwilson** [comment]
@jianyuan can you please clarify if custom filters are supported in v0.12.3? I can't see any reference to it in the docs. Thanks!
